app/actions/app_feature_update.rb

```diff
@@ -6,8 +6,10 @@ def self.update(feature_name, app, message)
         app.update(enable_ssh: message.enabled)
       when 'revisions'
         app.update(revisions_enabled: message.enabled)
-      when 'file-based-service-bindings'
-        app.update(file_based_service_bindings_enabled: message.enabled)
+      when 'service-binding-k8s'
+        app.update(service_binding_k8s_enabled: message.enabled)
+      when 'file-based-vcap-services'
```

Comment:
Could you use the constants introduced below (FILE_BASED_SERVICE_BINDINGS_FEATURE)?

Reply:
good point!